README.md

```diff
@@ -134,6 +135,13 @@ module.exports = {
         options: {
           name: '[name].[ext]?[hash]'
         }
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'vue-style-loader',
+          'css-loader'

```

Comment:
Had issues with this  and had do to npm install for it, should there be an entry to include this in the package.json as well?

Reply:
@saiyijinprince I added those lines to `webpack.config.js` and run `npm run build` and go no problem.